src/components/modules/api/blocks.ts

```diff
@@ -229,4 +230,22 @@ export default class BlocksAPI extends Module {
       'Use blocks.insert() instead.', 'warn');
     this.insert();
   }
+
+  /**
+   * Toggles BlockSetting of Current Elements
+   *
+   */
+  public toggleBlockSettingByIndex(): void {
```

Comment:
I would suggest to add an optional `state = true` param. If `true`, it'll open, if `false` it'll close.

Reply:
Should I put the default value of `state = true`?